kube_proxy/tox.ini

```diff
@@ -1,20 +1,26 @@
 [tox]
 minversion = 2.0
-basepython = py27
+basepython = py37
 envlist =
-    {py27,py36}-unit
+    py{27,37}-unit
```

Comment:
Our ci won't support 3.7 until https://github.com/DataDog/integrations-core/pull/2973 is fixed